eq_translations/extractable_strings.py

```diff
@@ -239,6 +239,72 @@
         "context": "Question",
         "additional_context": ["ListHeading", "ListDescription"],
     },
+    {
+        "json_path": "$..primary_content[*].title",
+        "description": "Introduction main title",
+    },
+    {
+        "json_path": "$..primary_content[*].contents[*].list[*]",
+        "description": "Introduction main list item",
+        "context": "PrimaryContent",
+        "additional_context": ["ListHeading", "ListDescription"],
+    },
+    {
+        "json_path": "$..primary_content[*].contents[*].description",
+        "description": "Introduction main description",
+        "context": "PrimaryContent",
+    },
+    {
+        "json_path": "$..primary_content[*].contents[*].guidance.contents[*].title",
+        "description": "Introduction main guidance title",
+        "context": "PrimaryContent",
+    },
+    {
+        "json_path": "$..primary_content[*].contents[*].guidance.contents[*].description",
+        "description": "Introduction main guidance description",
+        "context": "PrimaryContent",
```

Comment:
ListHeading should be additional

Reply:
Added `ListHeading` as additional. Thanks!